package.json

```diff
@@ -7,17 +7,18 @@
   "main": "dist/imgix-core-js.cjs.js",
   "module": "dist/imgix-core-js.esm.js",
   "browser": "dist/imgix-core-js.umd.js",
-  "type": "module",
```

Comment:
Why was this removed? It seems necessary to support ESM for consumers.

Reply:
It was removed because it caused conflicts with testing esm code with mocha. 

It seemed necessary to me as well, but it also seems to cause more trouble than what it's worth. Also we produce cjs, umd, and esm code here. Doesn't that support our ESM consumers?

If you know of a way of specifying `"type": "module"` in a way that doesn't conflict with our current setup and/or if specifying the type is _actually necessary_ lmk. I haven't found anything, but maybe I'm looking in the wrong places. Thanks!

Reply:
What were the conflicts exactly?

Reply:
Mocha gets confused. It either wants `mjs` extensions everywhere and `"type": "module"` or `js` extensions (omitted i.a.w. @frederickfogerty recommendation) but without `"type": "module"` (and using the esm package).

IMO I prefer using the `.mjs` extensions explicitly (just seems a little cleaner/explicit). I made a PR for that solution as well. Right now I'm leaning towards PR #209 what do you think? @sherwinski